packages/account/src/Components/address-details/__tests__/address-details.spec.tsx

```diff
@@ -32,10 +33,10 @@ describe('<AddressDetails/>', () => {
     const verification_info =
         'We need this for verification. If the information you provide is fake or inaccurate, you won’t be able to deposit and withdraw.';
 
-    let modal_root_el;
+    let modal_root_el: any;
 
-    const mock_props = {
-        fetchStatesList: jest.fn(() => []),
+    const mock_props: TAddressDetails = {
```

Comment:
```suggestion
    const mock_props: React.ComponentProps<typeof AddressDetails> = {
```
you can type it like this . then you don't need to expose `TaddressDetails` only for using it here.